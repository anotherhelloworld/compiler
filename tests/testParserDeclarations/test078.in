type
    TPoint = record
        x: real;
        y: real;
    end;
procedure P(a: integer; b: integer; Point: TPoint);
var
    NewPoint: TPoint;
begin
end;