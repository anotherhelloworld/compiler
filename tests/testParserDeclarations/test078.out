Type   TPoint
   Record
      Var   x
         Type   real

      Var   y
         Type   real

   End

Procedure   P   3
   Var   a
      Type   integer

   Var   b
      Type   integer

   Var   Point
      Record
         Var   x
            Type   real

         Var   y
            Type   real

      End

   Var   NewPoint
      Record
         Var   x
            Type   real

         Var   y
            Type   real

      End

   begin
   end

